plugins/woocommerce/includes/wc-formatting-functions.php

```
@@ -1189,16 +1189,25 @@ function wc_format_product_short_description( $content ) {
- * Formats curency symbols when saved in settings.
+ * Formats currency separators when saved in settings.
  *
- * @codeCoverageIgnore
+ * @since 10.8.0
  * @param  string $value     Option value.
- * @param  array  $option    Option name.
+ * @param  array  $option    Option data including 'id' and 'default'.
  * @param  string $raw_value Raw value.
  * @return string
  */
 function wc_format_option_price_separators( $value, $option, $raw_value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)